packages/frontity-org-theme/src/components/styles/homepage.tsx

```diff
@@ -38,23 +39,14 @@ const homePageStyles = (state: FrontityOrg["state"]["theme"]) => css`
   }
   /* Hero section */
   .hero-homepage {
-    background-color: #f2f3fc;
+    background-color: rgb(236, 237, 246);
+    height: 47em;
     position: relative;
-    clip-path: polygon(0 0, 100% 0, 100% 92%, 0% 100%);
-
-    &:after {
-      content: "";
-      display: block;
-      position: absolute;
-      background-color: rgb(236, 237, 246);
-      bottom: 0;
-      left: 0;
-      right: 0;
-      top: 0;
-      height: 100%;
-      width: 100%;
+    clip-path: polygon(0 0, 100% 0, 100% 88%, 0% 100%);
 
-      clip-path: polygon(0 100%, 100% 85%, 100% 92%);
+    @media only screen and (max-width: 769px) {
+      height: auto;
+      clip-path: none;
```

Comment:
Hey, you removed the `clip-path` for the mobile version. Maybe a different `clip-path` is needed but it should have one, according to the design.